.envrc

```diff
@@ -0,0 +1 @@
+use flake . -Lv --impure
```

Comment:
Why is this needed? Also note that this might clash for users who use `direnv` but not nix (flakes).

Reply:
Yeah - see all of this as a proposal. I can remove this if you don’t want it ;) 

some want it because it makes onboarding (for nix users at least) quite neat but some repos also have this in their gitignore. I guess with rust it’s already so trivial that this is not needed. 

I will remove it…

Reply:
Yeah, a good nix setup would be great, I just want to make sure everything works on non-nix setups as well.

Reply:
Sure. Yeah I will have to add some stuff to make this a “good nix setup” anyway, including some docs so perhaps I’ll remove it and it’ll be good enough :)